src/hooks/useServiceContainer.ts

```diff
@@ -150,20 +149,18 @@ export const useServiceContainer = <T extends TServiceData, U extends TServicePa
       .then((response) => {
         // In a future React version (potentially in React 17) this could be removed as it will be default behavior
         // https://stackoverflow.com/questions/48563650/does-react-keep-the-order-for-state-updates/48610973#48610973
-        ReactDOM.unstable_batchedUpdates(() => {
```

Comment:
Did you test `ReactDOM.unstable_batchedUpdates` is not needed anymore?

Reply:
Yes I verified it by using a test like this:

```  const [count, setCount] = useState(0);

  useEffect(() => {
    setTimeout(() => {
      setCount((c) => c + 1);
      setCount((c) => c + 1);
    }, 1000);
  }, []);

  console.log('Render count:', count);
```

also see article that mentioned this change: https://github.com/reactwg/react-18/discussions/21